web/angular-wallet/src/environments/environment.hmr.ts

```diff
@@ -9,5 +9,10 @@ export const environment = {
     tickerUrl: 'https://min-api.cryptocompare.com/data/pricemultifull?fsyms=BURST&tsyms=BTC,USD,EUR',
     tickerInterval: 60 * 1000
   },
+  performance: {
+    fcasUrl: 'https://platform-api.flipsidecrypto.com/api/v1/assets/573/metrics/FCAS?api_key=6e9c2808-0c48-41d8-81f1-5363d5ceb1af&change_over=7',
```

Comment:
hehe, I saw this on CMC. FCAS doesn't have an official API (asked them already)...the key is CMCs key :roll_eyes: